openslides_backend/action/actions/meeting/export_helper.py

```diff
@@ -55,7 +58,35 @@ def export_meeting(datastore: DatastoreService, meeting_id: int) -> Dict[str, An
         else:
             export[str(collection)] = {}
 
-    add_users(meeting.get("user_ids", []), export, meeting_id, datastore)
+    user_ids = set()
+    for id_ in meeting.get("user_ids", []):
+        user_ids.add(id_)
+    for collection in export:
```

Comment:
Instead of all this you could put the following lines after existing line 57, because all data is already read:
     
```
user_ids.update((result.get("user_id") for result in results[collection].values() if result.get("user_id")))
       for result in results[collection].values():
             user_ids.update(result.get("user_ids", ()))

```

Reply:
Done.